### PR TITLE
Mockable URL launcher

### PR DESCRIPTION
**Pull request by jpnurmi**
As suggested in #678, to avoid assumptions on url_launcher internals.

### CODE REVIEW

**Comment by jpnurmi** [comment]
>  92.30% of diff hit (target 93.64%)

argh, the test coverage bar is getting high :D

File: packages/ubuntu_wizard/lib/src/utils/url_launcher.dart [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Future<bool> launchUrl(String url) {
+  return getService<UrlLauncher>().launchUrl(url);
+}
+
+class UrlLauncher {
+  Future<bool> launchUrl(String url) async {
+    if (!await canLaunch(url)) {
+      log.error('Unable to launch $url');
+      return false;
+    }
```
Comment (oSoMoN) [inline packages/ubuntu_wizard/lib/src/utils/url_launcher.dart:17]: Would it be useful to log a debug message when launching a URL ?

**Review by oSoMoN — APPROVED** [review]
LGTM. Just one very minor suggestion.

File: packages/ubuntu_wizard/lib/src/utils/url_launcher.dart (hunk 1 above)
Comment (jpnurmi) [reply packages/ubuntu_wizard/lib/src/utils/url_launcher.dart:17]: Sure, could be useful :+1:

**Pull request merged**